### PR TITLE
Add cors via rest

### PR DESCRIPTION
**Pull request by Tharsanan1**

### CODE REVIEW

File: en/docs/create-api/create-and-attach-api-policies/cors/enable-cors-via-rest-api.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+   corsConfiguration:
+     corsConfigurationEnabled: true
+     accessControlAllowCredentials: true
+     accessControlAllowOrigins:
+     - "*"
+     accessControlAllowHeaders:
+     - authorization
+     accessControlAllowMethods:
+     - GET
+     accessControlExposeHeaders:
```
Comment (tharindu1st) [inline en/docs/create-api/create-and-attach-api-policies/cors/enable-cors-via-rest-api.md:92]: Add Access Control max age as well

**Review by CrowleyRajapakse — DISMISSED** [review]

**Comment by Tharsanan1** [comment]
[cors.pdf](https://github.com/wso2/docs-apk/files/12117458/cors.pdf)

File: en/docs/create-api/create-and-attach-api-policies/cors/enable-cors-via-rest-api.md (hunk 1 above)
Comment (Tharsanan1) [reply en/docs/create-api/create-and-attach-api-policies/cors/enable-cors-via-rest-api.md:92]: [cors.pdf](https://github.com/wso2/docs-apk/files/12143379/cors.pdf)
Done

**Comment by Tharsanan1** [comment]
Updated doc
[cors.pdf](https://github.com/wso2/docs-apk/files/12143395/cors.pdf)

**Review by CrowleyRajapakse — APPROVED** [review]

**Pull request merged**